### ISSUE TITLE
[ Feature Request ] Add possibility to download playlists

### ISSUE THREAD

**Issue by SindreKjelsrud** [kepelet/flo#28]
As of now it's not possible to download custom made playlists in Navidrome for offline use in _**flo**_. This would be a nice feature to have 😊

**Comment by faultables** [comment]
yeah it takes a _small_ adjustment considering the way flo downloads songs is tied tightly to albums (my bad). after that we can download songs by album (current), single track, artist, and eventually playlist!

**Comment by faultables** [comment]
Resolved in 1.6!
